app/Kconfig

```diff
@@ -7,4 +7,21 @@ config APP_DATA_LOGGING_SYNC_INTERVAL
 		This configures the interval at which the logging system flushes everything to disk
 		and hence determines the maximum duration of data lost on power cuts.
 
+choice
+	prompt "IMU"
+	help
+		This configures Eurus Flight Controller primary IMU.
+
+config APP_DATA_USE_MPU6050
+ 	bool "MPU6050"
+	help
+		Eurus Flight Controller shall use MPU6050 as primary IMU.
+
+config APP_DATA_USE_ICM42688P
+	bool "ICM42688P"
+	help
```

Comment:
All of these help strings feel a bit redundant and unnecessary IMO, at that point the user already knows this info. If we have some more context we can add it here tho,